presto-main/src/test/java/com/facebook/presto/block/TestArrayBlock.java

```diff
@@ -25,6 +25,8 @@
 
 import static com.facebook.presto.spi.type.BigintType.BIGINT;
 import static com.facebook.presto.spi.type.VarcharType.VARCHAR;
+import static org.testng.Assert.assertEquals;
```

Comment:
It would be nice to add some motivation for this change to commit message.
Is that common that block builder is created and a value is never appended to it?